docs/sources/administration/data-source-management/teamlbac/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+aliases:
```

Comment:
It's slightly confusing to have this specified but with an implicit `null` value.
Probably less confusing to just remove it.

```suggestion
```